src/Movim/ImageSize.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Movim;
+
+enum ImageSize: string
```

Comment:
Does it actually require a specific file for that ? 

Having the enum as a static parameter in Movim/Image should do it no ?